### ISSUE TITLE
[BUG] Uncaught SyntaxError: await is only valid in async function

### ISSUE THREAD

**Issue by kimsk** [dkackman/chia-repl#15]
I try [Connecting and Calling a Node Function](https://github.com/dkackman/chia-repl#connecting-and-calling-a-node-function) but got the error, `Uncaught SyntaxError: await is only valid in async function`

```javascript
chia-repl/src on  main is 📦 v0.11.1 via  v14.17.4
❯ npm start

> chia-repl@0.11.1 start
> node index.js

🌿 Welcome to Chia!
Type .help or .more-help to get started
🌿 .connect
Connecting to wss://localhost:55400...
Connected
🌿 await chia.full_node.get_network_info()
await chia.full_node.get_network_info()
^^^^^

Uncaught SyntaxError: await is only valid in async function
```

- OS: **WSL Ubuntu 20.04**
- Chia version **1.4.0**
- Version **from source; last 5 commits are shown below**

```bash
❯ git log --oneline -n 5
e757c74 (HEAD -> main, origin/main, origin/HEAD) Merge pull request #13 from dkackman/dependabot/npm_and_yarn/src/jshint-2.13.5
c0a8c3f Bump jshint from 2.13.4 to 2.13.5 in /src
95cbaa8 move editorconfig
28e46da create settings files if not found
98d1393 Merge branch 'main' of https://github.com/dkackman/chia-repl
```

**Comment by dkackman** [comment]
I cannot reproduce that. Does it happen with any rpc call?

What version of node do you have installed? `node --version`
Top level await was added in a later release.

**Comment by kimsk** [comment]
Thanks! It's been a while since I touch JS or TS. My node version was `v14.17.4`. 

This works now with node `v16.16.0`!

FYI, I got hard error, `Error while loading BLS module`, when using `v18.5.0`. 

![image](https://user-images.githubusercontent.com/116863/178511103-52d6a8ab-27b4-411e-a81b-021368eb719e.png)

**Comment by dkackman** [comment]
> Error while loading BLS module, when using v18.5.0.

I stick with the LTS version for everything, so not super surprised. Keep the feedback coming. Hope you find this useful!

**Comment by dkackman** [comment]
This fix is published in version 0.11.3. https://www.npmjs.com/package/chia-repl/v/0.11.3
